content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,207 @@
+---
+title: "Glossary"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a logical group of <<Workload>> defined by the <<Label App>>.
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<App Label>> on the <<Workload>>
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+link:https://istio.io/help/glossary/#deployment[Istio Glossary]
+
+== Destination
+
+The remote upstream service Envoy is talking to on behalf of a source workload. There can be one or more service versions for a given service and Envoy chooses the version based on routing rules.
```

Comment:
this is copied directly from istio docs, which means it is duplicate and will be hard to maintain. IMO if part of the glossary is identical to Istio, it is best to directly link there.

Reply:
same goes for several other objects mentioned here that are defined in Istio docs (Gateway, etc)

Reply:
Yeah, So should we only put the link to the documentation and add some lines with additional information for Kiali?